### ISSUE TITLE
Need a new state for cluster when restore is in progress.Currently it is shown as being in "Updating" state.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#18688]
Rancher server version - Latest build from master - Mar 6 - c8da22fb

Steps to reproduce the problem:
Create a DO cluster with 3 nodes.
Create few projects/workloads.
Take a backup.
Create few projects/workloads.
Restore cluster to backup.
Cluster state transitions to "Updating" when restore is in progress and then transitions to "Active" once restore is done.

Ideally we should have a state that reflects the fact that restore is in progress . "Updating" state could mean that any update is in progress.

We may also want to restrict certain actions from happening when the cluster is in process of restoring , like not allow for another restore to be started  or not allow another backup to be taken.

**Comment by alena1108** [comment]
We can introduce a new state based on the restore being in progress for a better visibility. But I would recommend against limiting certain operations when cluster is in restoring state. Because you might still want to be able to be able to add new etcd nodes and remove old ones (currently covered by update), or to pick another backup to restore to to correct situation if restore gets stuck by some reason. As there is really no rollback from restore once in progress, especially when it was called when the cluster was already in trouble, so whatever you need to do to correct operation must be done from this state.

As long as RKE can handle it correctly - and I believe it does by not proceeding with a cluster reconcile till etcd is up - we should be fine.

@moelsayed please confirm.

**Comment by moelsayed** [comment]
@alena1108 That is correct. REK will not proceed until etcd cluster is properly up. I also agree on the point of not disabling actions during restore.

**Comment by cjellick** [comment]
moving out. not a blocker.
